core/src/main/java/io/questdb/griffin/engine/functions/bind/ByteBindVariable.java

```diff
@@ -62,4 +65,9 @@ public boolean isThreadSafe() {
     public void toPlan(PlanSink sink) {
         sink.val("?::byte");
     }
+
+    @Override
+    public void toSink(@NotNull CharSink<?> sink) {
+        sink.put(value).put("::byte");
```

Comment:
some of the `<Type>BindVariable.toSink()` methods handles the `NULL` value, others (such as this one) do not.
should all of them check for `NULL`?